client/disputes/utils.ts

```diff
@@ -58,6 +61,10 @@ export const isAwaitingResponse = (
 	return disputeAwaitingResponseStatuses.includes( status );
 };
 
+export const isUnderReview = ( status: DisputeStatus | string ): boolean => {
+	return disputeUnderReviewStatuses.includes( status );
+};
+
 export const isInquiry = ( dispute: Dispute | CachedDispute ): boolean => {
```

Comment:
Not a part of this PR, but I see that `isInquiry` accepts a `Dispute` object. This is inconsistent with `isAwaitingResponse` and `isUnderReview` that accept `DisputeStatus`.

IMO we should change `isInquiry` to accept `DisputeStatus` to be consistent with the other util functions since the function doesn't need any other dispute details.

Reply:
I'll create a separate issue for this.

Reply:
https://github.com/Automattic/woocommerce-payments/issues/7085